### PR TITLE
GoofSec 1.5: Departmental Batons, Departmental Guard Offices, Batons from Backpacks -> Belts, Guard Cuffs -> Cableties, Mapping Icons for Jobs, Belt Reskinning, Brig Officer Baton Change, Sec Medic Gun Removal + Medical Access Removal, Security Sergeant Removal

### PR DESCRIPTION
**Pull request by Iamgoofball**
## About The Pull Request

All maps now have proper Departmental Guard offices, instead of Security outposts in each department. The computers in said offices have been adjusted where needed.
Departmental Guards now have lockers that contain two sets of departmental guard gear in their new offices.
Departmental Guards now have Departmental Batons. 
Departmental Batons work as normal inside their department.
Departmental Batons have three charges for non-departmental uses.
A charge is expended when you baton someone outside of your department.
If all three charges are expended, the baton is unusable until you return to your department and reactivate the baton.
This will reduce the ability of Guards to roll around the station and exert authority where they aren't supposed to be exerting authority.
This will encourage Guards to naturally stay near their department and keep it safe.
The charges system allows Guards to punish people who kite around the edge of their department, while punishing Guards who over-extend too far and should be calling Security.

Spawn points have been added to all maps for Departmental Guards.

Departmental Guards have had their cuffs replaced with colored cableties for their department.
This will encourage bringing folks to the front of the department and booting them out, rather than detaining and holding them.

Departmental Guards are given security HUDs, not HUDglasses, primarily so that they can see job icons and see if someone who ran into their department followed by redshirts is wanted by Security or not, so as to improve communication.
They do not have the access needed to adjust records, however, and will be on read-only mode.

All Departmental Guard, Security Officer, HoS, and Warden Stun Batons are now in the belts in your security lockers, instead of in your backpack.
This is to keep the number of batons on the map at a consistent level regardless of Security staffing.
Your Security Belt can now be reskinned between Security and Peacekeeper variants.

Brig Officers have had their baton replaced by a Departmental Prison Baton.
Brig Officers have had their taser removed and replaced with a disabler because it's not 2012 anymore and we don't give Security tasers.

Security Medics have had their telescopic batons removed with a normal stun baton in its place in their locker, because that's a Head of Staff item and shouldn't be available to rank and file staffers. Telescopic baton creep is a problem and is being slowly addressed.
Security Medics have had their firearm removed and replaced with a Disabler, as it was an oversight from the original Goofsec PR, and they shouldn't be packing heat like that, especially if they aren't supposed to be making arrests.
Security Medic has had its access to Medical removed. This resolves issues with Security Medics going into Medical and looting it of supplies and vendor equipment round start.

Arguably, Security Medic is a terrible role that existed only to give Security a way to powergame safe medical treatment, but removing it is not for this PR.

Map icons have been added for the spawn points of all new Skyrat jobs.

Security Sergeant has been removed after discussion with policymakers and other developers on the project.

The Sarge as a job simply was too much extra hierarchy in an already small department. On-the-fly commanding is generally done by the HoS already, and the 2nd in command post is handled by the Warden.

It has been replaced with an extra Security slot.

## How This Contributes To The Skyrat Roleplay Experience

Please see the above section.

## Changelog
:cl: Iamgoofball + whoever made the contractor baton sprites I codersprite modified
add: All maps now have proper Departmental Guard offices, instead of Security outposts in each department. The computers in said offices have been adjusted where needed.
expansion: Departmental Guards now have lockers that contain two sets of departmental guard gear in their new offices.
add: Departmental Guards now have Departmental Batons. 
balance: Departmental Batons work as normal inside their department.
balance: Departmental Batons have three charges for non-departmental uses.
balance: A charge is expended when you baton someone outside of your department.
balance: If all three charges are expended, the baton is unusable until you return to your department and reactivate the baton.
balance: This will reduce the ability of Guards to roll around the station and exert authority where they aren't supposed to be exerting authority.
balance: This will encourage Guards to naturally stay near their department and keep it safe.
balance: The charges system allows Guards to punish people who kite around the edge of their department, while punishing Guards who over-extend too far and should be calling Security.
balance: Departmental Guards have had their cuffs replaced with colored cableties for their department.
add: Departmental Guards are given security HUDs, not HUDglasses, primarily so that they can see job icons and see if someone who ran into their department followed by redshirts is wanted by Security or not, so as to improve communication.
balance: They do not have the access needed to adjust records, however, and will be on read-only mode.
del: All Departmental Guard, Security Officer, HoS, and Warden Stun Batons have been removed from the backpack, and are now in the belts in your security lockers, instead of in your backpack.
balance: This is to keep the number of batons on the map at a consistent level regardless of Security staffing.
qol: Your Security Belt can now be reskinned between Security and Peacekeeper variants.
balance: Brig Officers have had their baton replaced by a Departmental Prison Baton.
balance: Brig Officers have had their taser removed and replaced with a disabler because it's not 2012 anymore and we don't give Security tasers.
del: Security Medics have had their telescopic batons removed, because that's a Head of Staff item and shouldn't be available to rank and file staffers. Telescopic baton creep is a problem and is being slowly addressed.
del: Security Medics have had their firearm removed, as it was an oversight from the original Goofsec PR, and they shouldn't be packing heat like that over other officers.
fix: Map icons have been added for the spawn points of all new Skyrat jobs.
fix: Spawn points have been added to all maps for Departmental Guards.
del: Security Sergeant has been removed after discussion with policymakers and other developers on the project.
add: An additional Security Officer slot has been opened.
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm
Comment (axietheaxolotl) [inline modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm:8]: Security area is still blue colour coded. Having bright red lockers is an eye sore, and will make them look bad.

File: modular_skyrat/modules/brigoff/code/brigoff.dm
Comment (axietheaxolotl) [inline modular_skyrat/modules/brigoff/code/brigoff.dm:58]: If you're gonna remove their taser, give them a disabler.

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (axietheaxolotl) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:66]: Same with what you did before, give them something to supplement this if you're gonna remove it. The SecMed has authority to act like an officer if need be, they just shouldn't be. Removing any tools for self-defence, or to make arrests is just dumb.

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (axietheaxolotl) [inline modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:68]: You did this before, but I didn't get around to mentioning it, there's no real reason for this. BlueSec should really be the default, as it's a Skyrat thing. Right now, we have an issue where there's a mix-match of security running around, and it just looks bad.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm
Comment (axietheaxolotl) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: I don't really like this change. Yeah, being able to change it into the peacekeeper belt aesthetically is nice. But the Peacekeeper belt had features the normal sec belt didn't have. Most notably the holster mechanic, which was really nice and satisfying. With this you're basically removing this feature.

**Review by axietheaxolotl — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:68]: You can alt-click your equipment to reskin it how you wish.

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm
Comment (Iamgoofball) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: The Holster mechanic was designed for a Security team that was packing pistols 24/7. They no longer do, so they no longer need it.

File: modular_skyrat/modules/brigoff/code/brigoff.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/brigoff/code/brigoff.dm:58]: Sure.

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:66]: I will give them a disabler. They will need to go to the locker for a baton, however.

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (axietheaxolotl) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:68]: I know that much. My main problem is how RedSec is now the default. Instead of Bluesec, for like, no reason.

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm:8]: I'm going to be talking with mappers about resolving the issues with the map coloring of Security not following the standard color-coding soon, as it's holding us back from getting the new and improved brig and permabrig for multiple maps at this rate. It's just a matter of whether I need to make the change myself or if I can get some assistance in doing so.

File: modular_skyrat/modules/goofsec/code/department_guards.dm
Comment (ghost) [inline modular_skyrat/modules/goofsec/code/department_guards.dm]:
```suggestion
					say("[src] is out of non-departmental charges! Return to your department and reactivate the baton to recharge it!")
```
(Personally would word it like this, by no means is this a required change)

File: modular_skyrat/modules/goofsec/code/department_guards.dm
Comment (ghost) [inline modular_skyrat/modules/goofsec/code/department_guards.dm]:
```suggestion
	desc = "A stun baton fitted with a departmental area-lock - outside of its department, it only has three charges."
```
(Personally would word it like this, by no means is this a required change)
This change would ideally apply to all the batons' descriptions

**Comment by ghost** [comment]
I cant leave this as a suggestion on a .dmi so instead I'll just post it here with screenshots:
https://gyazo.com/d10a7c36b2d6ae1c1420bd719485db1f
These two _old icons are almost exactly the same as the ones you've copied down with our newsec stuff; on top of that, you probably dont need to move the upstream icons down from their original position, keep them on their side of the SKYRAT icon-state-marker-thing (to help maintainers know where to just blindly copy-paste)

also you can remove the 'mark oldsec as _old' icon-state-note-thing now that we're going back to red HUD icons

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (thestubborn) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:68]: the reason is bluesec sucks

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm
Comment (thestubborn) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: wouldnt the disabler count as a pistol?

File: modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm
Comment (thestubborn) [reply modular_skyrat/modules/sec_haul/code/security_medic/security_medic.dm:68]: just because something is a "skyrat thing" doesn't mean its a good thing

**Comment by YakumoChen** [comment]
Since you're reverting to redsec (based) you're free to remove the clothing crate I added for redsec

File: code/modules/admin/verbs/mapping.dm
Comment (Gandalf2k15) [inline code/modules/admin/verbs/mapping.dm:318]: Properly format the edits, marking the start and the end

Comment (Iamgoofball) [reply code/modules/admin/verbs/mapping.dm:318]: whoops, forgot to undo this

Comment (Iamgoofball) [reply code/modules/admin/verbs/mapping.dm:318]: proper start/end markers placed

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm
Comment (YakumoChen) [reply code/game/objects/structures/crates_lockers/closets/secure/security.dm]: disablers are not pistols, pistols are a specific subcategory of ballistic weapons in the code.
The PK belt only holds pistol-type weapons, and I think all of the ones that fit in the belt are removed from the game.

File: modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm
Comment (YakumoChen) [inline modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm:363]: remove the whole gunset box since it only applies to sarges anyway.

**Review by YakumoChen — CHANGES_REQUESTED** [review]

File: modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm:363]: I figured I'd leave it in if anyone needed it for stuff, but I suppose I can do that too.

Comment (YakumoChen) [reply modular_skyrat/modules/sec_haul/code/guns/token_system/gunsets.dm:363]: you removed the only reasonable way to obtain the box so it's not really needed.

**Comment by dawsonkeyes** [comment]
I used the disabler. You know why? Cause the disabler doesn't lethal, and unlike the shitty 12 gauge it stops a criminal in their tracks in three hits. Bang, bang, bang, and they're fucking done. I use four shots just to make damn sure. Because, once again, I'm not there to coddle a buncha criminal scum sucking fuckers, I'm there to 1) Survive the fucking round. 2) Guard the armory. So you can absolutely get fucked. If I get veteran whitelist, which I won't, you can guarantee I will continue to use the disabler to apprehend criminals. Because it's quick, clean and effective as fuck. Why in the seven hells would I fuck around with the baton shots, which take half a battery just to bring someone down, or with the shotgun slugs which are slow as balls, impossible to aim and do about next to jack shit, fuck all. The disabler is the superior law enforcement weapon. Because it stops crime. And it stops crime by reducing the number of criminals roaming the fucking halls.

File: modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm
Comment (Iamgoofball) [reply modular_skyrat/modules/sec_haul/code/peacekeeper/peacekeeper_lockers.dm:8]: We got changes coming to the basic color palette of bluesec after talking with Xyel and a spriter. I'll keep ya posted.

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (RatFromTheJungle) [inline modular_skyrat/modules/goofsec/code/projectiles.dm]: this buffs the PCR  to a time to down of around 2s, making it double a bulldog with buckshots (5s from 2 tiles), making it un-comparably the best current firearm on station.

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (RatFromTheJungle) [inline modular_skyrat/modules/goofsec/code/projectiles.dm]: this buffs the Pitbull to a time to down of around 3.5s above its current of 5s (equal to a bulldog with buckshot, at about 2 tiles)

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (RatFromTheJungle) [inline modular_skyrat/modules/goofsec/code/projectiles.dm]: this thing is still just terrible, average of 9s to down (compared to a bulldog with buckshots 5)

all following are done on local and live, vs unarmored humans

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (RatFromTheJungle) [inline modular_skyrat/modules/goofsec/code/projectiles.dm]: this makes the ostwind down in about 5s, equal to the bulldog with buckshot from 2 tiles

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (RatFromTheJungle) [reply modular_skyrat/modules/goofsec/code/projectiles.dm]: these dropped in the wrong order, but they were tested on local and live, all 4
local both with this branch, and current master
live server being well, the live branch

File: modular_skyrat/modules/goofsec/code/projectiles.dm
Comment (dawsonkeyes) [reply modular_skyrat/modules/goofsec/code/projectiles.dm]: this is the same damage as an energy sword but also you can shoot it through windows and security has it

**Comment by CandleJaxx** [comment]
i'd suggest retaining medsec's access to medical, given how they're listed in 'medical' and not 'sec' in the manifest

**Comment by Lauwma** [comment]
We shouldn't be removing buckshot because in the Skyrat ecosystem it's not used the same way it's used in the tgstation ecosystem, the relevant thread that you quoted is not an enough argumentation to remove it since its conclusion is highly questionable logic wise. Just because someone tells you something is bad doesn't mean it's bad, you need pragmatic arguments for it to be bad. 

If we want pragmatic arguments i'm asking you all when was the last time you saw buckshots ? Yes, probably a long time ago, and probably not used by security but an antagonist. 
In fact buckshots, for security, are already arguably bad in the situations it should be used in, let's look why.

Buckshots are the only lethal option on a shotgun, meaning that's the only CQC gun ammunition they have. But security are not encouraged to use lethals most of the times, buckshots are not used against the antagonists they are relevant against, traitors. Why so ? Because a traitor you can baton them or disable them, which is more efficient, will not get you bwoinked, and is also rp friendly.
But what about lings ? Well lings in the current ecosystem don't care about buckshots and shrugs the wounds with their funny regenerate button that only takes 10 chemicals. (Yes removing broken legs, bleeding, ect for only 10 chemicals).

What about mobs ? The principal strength of buckshot is their wound chance. Mobs don't care about wound chances. Therefore buckshot is bad against mobs. Same goes for blobs and molds.

What about nukies ? Well, buckshot is arguably good against nukies indeed, because it allows you to break the hardsuits shields and bypass the dsword laser immunity. Though we don't see nukie much and we don't see buckshot being used anyway because there's simply better options like the cargo autorifles.

What about wizards, well it depends on the wizard but once again there's better options than the buckshot, that are more rp friendly and won't get you bwoinked.

In conclusion security players are not using buckshots because it's not appropriate for the situation it needs to be used in. Buckshot shouldn't be removed since it would remove sec only lethal CQC option (that they should use more) and get it's value tweaked like lowering the wound chances in trade off for more damages (since they are to be mainly used against mobs or things that don't care about wounds).

**Comment by ThinkArbies** [comment]
Most of this PR is bloated with so many different changes that have nothing to do with Department Guards at all, so I'm not even sure why all these unnecessary sec changes are lumped in when the hot topic of discussion was changing (or preferably removing) department guards to prevent the current abuse of the role.

So far I see department guards being given tools that allow them to continue to perpetuate their abuse, albeit with paper-thin limitations. What does "reactivating" their stunbaton mean in this instance? Do they have to charge it like with normal batons? Or do they simply have to interact with it the moment they step back into their department?

Furthermore, why give them security HUDs in the first place? Their job isn't to question why security is conducting what should, ideally, be a lawful arrest. It's to keep troublemakers out of their departments and protect property. If redshirts are chasing someone that should be reason enough to open the doors and take a step back. They don't need a flashing red 'A' to tell them what context clues should already. What's going to happen is that the current slew of dep guard players are just going to use the HUD to alert people to when they've been set to arrest, thus forcing Security to avoid setting arrest statuses for fear of the suspect catching wind and getting away or hiding the evidence of any wrongdoing.

The big thing about Dep Guards was to discuss policy changes, not change the tools at their disposal. This won't stop people from abusing the role, nor does it go to any lengths to define the sort of relationship Dep Guards should be having with Security as a whole.

As for the Sec changes, as mentioned above these strike me for the most part as both entirely unnecessary and given the context of how dep guards have been your pet project - perhaps a little retaliatory - especially when they're hamfisted into this PR for no reason at all. The changes to Sec should be separated and placed into an alternate PR where they can be given proper reviews, rather than try to trick devs into merging what should rightly be two different PRs under disguise of one.

I'll summarize my view of the bulk of these changes.

- I have no strong opinion on keeping sec sergeant or removing it. It's true, it's hardly used as it should and most of the field coordination is done by the HoS anyways. However, allow the HoS to keep the sergeant token and allow them to use it to give an officer some extra authority if they please.

- Giving COs and Sec Medics disablers in place of their normal taser/pistol is good.

- Replacing CO batons with what is just the department guard's baton is... Questionable. On code red, COs are permitted **and expected when necessary** to leave the Brig and act in the capacity of a normal officer. Gimping their baton gets in the way of that. And I personally haven't encountered instances where COs were unnecessarily acting like regular officers and mucking around outside the Brig trying to arrest people.

- The Security Medic changes are probably the most egregious in this whole PR. They've already been gimped hard with recent limitations on what they can and can't do. Removing their medical access just relegates them to a sec officer with a basic medkit. At this point they **should** just be removed and replaced with an extra officer slot, because right now they serve a very redundant purpose that anyone could fulfill by picking up the medkit from the Brigbay. Secmeds have become an RP role at best, but even then the tools to facilitate said RP is being taken away from them slowly but surely. That bit about them being a 'terrible role' is anecdotal at best. 

"Arguably, Department Guards are a terrible role that existed only to give shitters a way to protect their metacliques and grief Security."

See?

- Reworking armory rifle spawns **yet again** serves to nullify the original intent behind their rework to begin with, and I don't even know why this change is being sandwiched in here because no actual reason for it is being given. It feels like it was added as an after-thought.

- The same goes for buckshot, since no one really uses it anyways. I don't particularly mind if the broken "automatic" shotgun (it's not actually automatic) and the magfeds are removed, since they've been broken for months and I reported that fact multiple times when they were a new addition and they still weren't fixed to begin with.

- Also, the PR being referenced for shotguns is **almost a year old and also a troll/salt PR** I don't know why this is being referenced or cited as current, up-to-date discussion when that PR has obviously been reverted at some point.

I'm going to go out and say it, even though the way it's being said might not be ideal.

**The security-half of this PR is so out of place that this is all very obviously a retaliatory 'gotcha' because the community voted so overwhelmingly against Department Guards and you don't like being strong-armed to make changes to your pet project.**

We'd all be better served by seeing this PR bisected so the department guard changes can be merged without unnecessary bloat that just gimps Security for no good reason.

**Comment by CandleJaxx** [comment]
i might also suggest putting all the non-departmental-guard changes in a separate pr so we can merge the guards and discuss sec stuff in discuss channel

**Comment by coffeecoffeecoffee** [comment]
strongly feel this needs to be divided up into many different PRs, as some of these are beneficial and some of these are in my opinion really awful. you can't tack sweeping security changes onto much-needed fixes for your dep guards!

**Comment by thestubborn** [comment]
>     * Replacing CO batons with what is just the department guard's baton is... Questionable. On code red, COs are permitted **and expected when necessary** to leave the Brig and act in the capacity of a normal officer. Gimping their baton gets in the way of that. And I personally haven't encountered instances where COs were unnecessarily acting like regular officers and mucking around outside the Brig trying to arrest people.

person who made the guards here, I said to goof to do that
I did also say the departmental lock should come off on red alert

**Comment by Iamgoofball** [comment]
We'll be doing an atomisation before merge.

**Comment by Iamgoofball** [comment]
@ThinkArbies

>So far I see department guards being given tools that allow them to continue to perpetuate their abuse, albeit with paper-thin limitations. What does "reactivating" their stunbaton mean in this instance? Do they have to charge it like with normal batons? Or do they simply have to interact with it the moment they step back into their department?

Did you even read the description? This is a nerf to guards, lol

>Furthermore, why give them security HUDs in the first place? Their job isn't to question why security is conducting what should, ideally, be a lawful arrest. It's to keep troublemakers out of their departments and protect property. If redshirts are chasing someone that should be reason enough to open the doors and take a step back.

If Security can't be bothered to take the 0.2 seconds to mark someone as arrest when they're actively arresting them, that's Security's problem. The HUD allows Guards read-only access to the records for this purpose.

>The big thing about Dep Guards was to discuss policy changes, not change the tools at their disposal. This won't stop people from abusing the role, nor does it go to any lengths to define the sort of relationship Dep Guards should be having with Security as a whole.

This actually does fix the primary issue, guards being able to be effective outside of their department and guards not having a way to tell if Security wants someone.

>Security to avoid setting arrest statuses for fear of the suspect catching wind and getting away or hiding the evidence of any wrongdoing.

I'm choosing to ignore this Info Wars tier conspiracy posting.

>trick devs into merging what should rightly be two different PRs under disguise of one.

You have no idea what you're talking about, and this makes it incredibly clear.

>Removing their medical access just relegates them to a sec officer with a basic medkit.

That is what Security Medic is, yes. I'm sorry you're upset you can't have Security Medics raid medical for you anymore for supplies. By what Cobalt has told me, Security Medics have everything they need to do their job in their medical bay on the maps. If this is not the case for an individual map, we can update said map to include more supplies for them.

>The security-half of this PR is so out of place that this is all very obviously a retaliatory 'gotcha' because the community voted so overwhelmingly against Department Guards and you don't like being strong-armed to make changes to your pet project.

Once again, you have no idea what you're talking about. Good thing I am allowed to ignore you.

**Comment by Iamgoofball** [comment]
@Lauwma 
For the record, we've got a method of "shotgun" style spread coming with Microfusion guns. Ideally I want the shotguns shifted more towards specialist shells(ie. the techshells) and general damage dealing to be the job of the MF guns.

**Comment by Iamgoofball** [comment]
>Reworking armory rifle spawns yet again serves to nullify the original intent behind their rework to begin with, and I don't even know why this change is being sandwiched in here because no actual reason for it is being given. It feels like it was added as an after-thought.

It was done to resolve a issue with armory weapon distribution being effectively random. That randomization actually broke recently, and was actually split off into a different PR.

**Comment by Iamgoofball** [comment]
>Buckshots are the only lethal option on a shotgun

For the record, this is incorrect. Multiple other shell types are available.

**Comment by Lauwma** [comment]
> >Buckshots are the only lethal option on a shotgun
> 
> 
> 
> For the record, this is incorrect. Multiple other shell types are available.

Well yes this is indeed incorrect because we have tech shells, my bad. Though tech shells needs to be prepared in advance and preparation is badly viewed by admins. You can't have your techshells ready for when you will need them. Buckshots are the cheap and ready to go lethal option that won't get you bwoinked. Issue with techshells is that security can't make them on their own even if scientists researched it, they need to ask for a million parts (the scientist will likely get it wrong, and sadly uses the crafting system.

Maybe if we had techshells printable for the same ressources cost they had when we had to craft them it could be a replacement for buckshot but I like techshells being the upgrade of buckshots, while buckshot is still ready to use in case you need it.

Removing buckshot also impacts antagonists, I forgot to mention it before. Antagonists that aren't in science can't make reliably techshells.

**Comment by ThinkArbies** [comment]
@Iamgoofball 

> If Security can't be bothered to take the 0.2 seconds to mark someone as arrest when they're actively arresting them, that's Security's problem. The HUD allows Guards read-only access to the records for this purpose.

I feel like redshirts chasing someone or asking after them would be clue enough. Once again Dep Guards have no business trying to stick their nose in security matters when they're conducting what should ideally be a lawful arrest, and anyone that does probably can, will and should be arrested in turn by any sane member of Sec, arrest status or no arrest status. If Dep Guards aren't expected to answer to the HoS, then security shouldn't be expected to answer to Dep Guards.

> The charges system allows Guards to punish people who kite around the edge of their department, while punishing Guards who over-extend too far and should be calling Security.

> This will encourage bringing folks to the front of the department and booting them out, rather than detaining and holding them.

These two points are contradictory at best. Say someone is right outside their department causing trouble and the guard goes to baton them.

...Then what? Do they drag them back in only to throw them out again? If they're not supposed to be detaining people, why are they expected to ply their baton on people outside their department as opposed to, you know, calling security the moment they step out of the department? The batons should be limited to the department **period** and cease working the moment they step outside the boundaries of said department. They shouldn't have the ability to gradually creep out of their department  for no real reason if they're not even supposed to be detaining people period. And since they shouldn't be, they shouldn't even have any form of cuffs to begin with because those are tools for doing exactly that.

> This actually does fix the primary issue, guards being able to be effective outside of their department and guards not having a way to tell if Security wants someone.

That is _hardly_ the issue at all. The issue is Department Guard players being unnecessarily combative and hostile to security and playing the role _solely_ to be exactly that. I just don't think the playerbase is ready for Department Guards at all and any benefits they could add to the server is non-existent thanks largely in part to the people that choose to play Dep Guard for all the wrong reason, and making code-based changes to how they play mechanically does nothing to alleviate the present LRP issue behind the people that play department guards or better define their policy **which they still don't have much of.**

**Comment by Iamgoofball** [comment]
I've been informed you're literally banned from every facet of communicating with the Skyrat community. I'm just gonna go ahead and mirror that ban to my blocklist, Thinkarbies, as you're not supposed to be commenting on this PR anyways.

**Comment by Zonespace27** [comment]
Small (possible) issue I noticed with the baton, it can't fit in pockets while folded up, which from what I've experienced means there's no reason to leave it folded up? Its in hand sprite (which is regular baton sprite) doesn't change, either.

**Comment by axietheaxolotl** [comment]
I still heavily believe this PR needs a lot of atomisation, and shouldn't really be merged in it's current state, where all of the changers are being forced in at once.

**Pull request merged**
